dev-python/virtualenvwrapper/virtualenvwrapper-4.8.4-r3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Set of extensions to Ian Bicking's virtualenv tool"
+HOMEPAGE="https://github.com/python-virtualenvwrapper/virtualenvwrapper
+	https://pypi.org/project/virtualenvwrapper/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/virtualenv[${PYTHON_USEDEP}]
+		dev-python/stevedore[${PYTHON_USEDEP}]
+		dev-python/virtualenv-clone[${PYTHON_USEDEP}]
+	')"
+BDEPEND="
+	${DEPEND}
+	$(python_gen_cond_dep '
+		dev-python/pbr[${PYTHON_USEDEP}]
+	')"
+
+PATCHES=(
```

Comment:
That's some heavy patching. Could you consider either using a snapshot instead, or pulling them over SRC_URI?

Reply:
A snapshot would work, all the test fixes were applied on the tree before any of the breaking changes. 

Pulling the patches with SRC_URI wouldnt be convenient A) adding the patches to SRC_URI individually isnt a great alternative B) I dont have a namespace on Gentoo infrastructure to host bundled patches, so I would to ask someone else for that.

Reply:
Snapshot would 20230115 at any commit in this range https://github.com/python-virtualenvwrapper/virtualenvwrapper/compare/c945ffbf7bb0f4aba707386c41f7789c9428b26c...c91d69e091b501a06cbe634e310785dcb4aceb51

Reply:
Yeah, let's do a snapshot. You know the package best, choose the commit you think would be best for this.